tests/pytorch_pfn_extras_tests/training_tests/extensions_tests/test_snapshot.py

```diff
@@ -62,7 +62,7 @@ def test_call():
 def test_savefun_and_writer_exclusive():
     # savefun and writer arguments cannot be specified together.
     def savefun(*args, **kwargs):
-        assert False
+        pytest.fail('never reach')
```

Comment:
note: flake8 does not allow `assert False` because of B011.
```
B011 Do not call assert False since python -O removes these calls. Instead callers should raise AssertionError().
```
flake8 suggests using `raise AssertionError(...)`, but I replaced this line with `pytest.fail` for the readability.